lib/bitcoin/lock/MongoDbLockTransactionStore.ts

```diff
@@ -26,7 +27,14 @@ export default class MongoDbLockTransactionStore {
    * Initializes this object by creating the required collection.
    */
   public async initialize () {
-    const client = await MongoClient.connect(this.serverUrl, { useNewUrlParser: true }); // `useNewUrlParser` addresses nodejs's URL parser deprecation warning.
+    // `useNewUrlParser` addresses nodejs's URL parser deprecation warning.

```

Comment:
I think this could be a good time to refactor all store class to inherit from base `MongoDbStore` so we can avoid all the duplicate code for instantiation.

The reason that this was not done earlier was just combination of  resourcing + `MongoDBStore` was introduced later than some of these store classes + 'don't touch classes if there is no need". So I opportunistically migrated some of these store classes overtime and it wasn't so bad from memory. It seems like the right time to do this for the remainder of the store classes.